README.md

```diff
@@ -9,6 +12,16 @@ then we create some `ViewHolder` to relate the type, all of the process it is a
 Today, I create a new graceful way to easily develop the complex RecyclerView/ListView, with my MultiType library, 
 no matter how complex and how frequently changing list, we could insert a new type without changing the old codes.
 
+## Getting started
+
+In your `build.gradle`:
+
+```groovy
+dependencies {
+    compile 'me.drakeet.multitype:multitype:1.0'
+}
+```
+
 ## Usage
 
 #### Step 1. Create a class extends `ItemContent`, for example:
```

Comment:
Ketsalot@googlemail.com and I will need the following models and each of the largest trade and the system link to the easy way for free in app purchase